benchmarks/README.md

```diff
@@ -316,8 +316,25 @@ The reports are generated at location "/tmp/benchmark/"
 ### Sample latency graph
 ![](predict_latency.png)
 
-# Benchmarking Apache Bench AWS
-If you're making a large change to TorchServe it's best to run an [automated benchmarking suite on AWS](https://github.com/pytorch/serve/tree/master/test/benchmark) so that you can test multiple CUDA versions and EC2 hardware configurations easily.
+# Auto Benchmarking with Apache Bench
+auto_benchmark.py runs Apache Bench on a set of models and generates report.md once [Apach bench installation](https://github.com/pytorch/serve/tree/master/benchmarks#installation-1) is done.
+```
+cd serve
+
+python benchmarks/auto_benchmark.py -h
+usage: auto_benchmark.py [-h] [--input INPUT] [--skip SKIP]
+
+optional arguments:
+  -h, --help     show this help message and exit
+  --input INPUT  benchmark config yaml file path
+  --skip SKIP    true: skip torchserve installation. default: false
+
+python benchmarks/auto_benchmark.py --input benchmarks/benchmark_config_template.yaml --skip true
+```
+
+- [benchmark_config_template.yaml](https://github.com/pytorch/serve/blob/master/benchmarks/benchmark_config_template.yaml) is a config template yaml file for benchmark automation.
+
+- Benchmark automation results are stored in local /tmp/ts_benchmark. /tmp/ts_benchmark/report.md is the final report. [Here](https://github.com/pytorch/serve/blob/master/benchmarks/report.md) is a sample final report.
```

Comment:
@lxing it will be good to clarify that this can be pushed to S3 if user has set up AWS credentials and pass the S3 bucket in the benchmark_config_template.yaml.